Docs: Git Sync - Remove GA notes (#127993)
grafana/grafana: docs/sources/as-code/observability-as-code/git-sync/git-sync-setup/_index.md

{{< admonition type="note" >}}

> Git Sync is under continuous development. [Contact Grafana](https://grafana.com/help/) for support or to report any issues you encounter and help us improve this feature.

Reviewer: ```suggestion
Git Sync functionalities are constantly evolving. [Contact Grafana](https://grafana.com/help/) for support or to report any issues you encounter and help us improve this feature.
```
